internal/hcsoci/network.go

```diff
@@ -1,28 +1,6 @@
 package hcsoci
 
-import (
-	"github.com/Microsoft/hcsshim/internal/hns"
-	"github.com/sirupsen/logrus"
-)
-
-func createNetworkNamespace(coi *createOptionsInternal, resources *Resources) error {
-	netID, err := hns.CreateNamespace()
-	if err != nil {
-		return err
-	}
-	logrus.Infof("created network namespace %s for %s", netID, coi.ID)
-	resources.netNS = netID
-	resources.createdNetNS = true
-	for _, endpointID := range coi.Spec.Windows.Network.EndpointList {
```

Comment:
Is docker on CD passing the `EndpointList`? I wonder if this logic was supposed to "upgrade a user of `EndpointList`" to a "`NetworkNamespace`" in RS5.